README.md

```diff
@@ -1,94 +1,46 @@
 ![](./resources/official_armmbed_example_badge.png)
-# Getting started with the Mbed OS block device API
+# Block Device API Mbed OS example
 
-This example demonstrates how to use the Mbed OS block device API.
+The example project is part of the [Arm Mbed OS Official Examples](https://os.mbed.com/code/). It contains an application that demonstrates how to use a block device as storage on supported [Mbed boards](https://os.mbed.com/platforms/).
```

Comment:
It's not necessary to link all official examples, we only do it for blinnky which is the "get started" example. In addition the first sentence (copied from blinky) is not the best! 
```suggestion
This example shows how to use a block device as storage on supported [Mbed boards](https://os.mbed.com/platforms/).
```